src/patternfly/components/Drawer/docs/code.md

```diff
@@ -0,0 +1,11 @@
+## Usage
+
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-drawer` | `<div>` | Creates the drawer container. |
+| `.pf-c-drawer__main` | `<div>` | Creates the drawer main container. |
+| `.pf-c-drawer__panel` | `<aside>` | Creates the drawer panel container. |
+| `.pf-c-drawer__panel-body` | `<div>` | Creates the drawer panel body container. |
+| `.pf-m-open` | `.pf-c-drawer` | Modifies the drawer for the open state. |
+| `.pf-m-inline` | `.pf-c-drawer` | Modifies drawer so the main element and panel element are displayed side by side. |
```

Comment:
I don't know if `inline` works here for the modifier name. Or maybe you need to update the description, because its more obvious in the demo then the component. I also wonder if these "inline" styles should be in the page component since they modify the drawer in the page component specifically.

Reply:
`inline` just displays the drawer inline with the content instead of on top of it. I'm open to another name if you want to suggest one. But they aren't just used in the page component. In fact, we don't support them in the page component at the moment. They would be supported in other contexts though, like [topology](https://patternfly-react.surge.sh/patternfly-4/components/topology/).

Reply:
Ok im good with that, I can't think of another name